airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -185,10 +230,16 @@ def check_connection(self, logger: logging.Logger, config: Mapping[str, Any]) ->
 
         customers = self.get_customers(google_api, config)
         logger.info(f"Found {len(customers)} customers: {[customer.id for customer in customers]}")
-
+        is_not_none = functools.partial(operator.is_not, None)
+        # Use only two accounts for validation: one manager account and one non-manager account
+        validate_customers = list(filter(is_not_none, [
```

Comment:
Is this just a performance optimization? Can you update the comment to explain why we only need one of each for validation?